include/vctr/Containers/Span.h

```diff
@@ -103,11 +103,34 @@ class Span : public VctrBase<ElementType, std::span<ElementType, extent>, extent
         VCTR_ASSERT (extent == containerToView.size() || extent == std::dynamic_extent);
     }
 
-    /** Copies the data of the source container to the memory this span refers to. */
+    //==============================================================================
+    // Operators
+    //==============================================================================
+    /** Copies or moves the data of the source container to this Span. You have to ensure that the source size matches. */
     template <has::sizeAndDataWithElementType<ElementType> Container>
-    constexpr Span& operator= (const Container& containerToCopyDataFrom)
+    constexpr Span& operator= (Container&& containerToCopyDataFrom)
+    {
+        if constexpr (has::begin<Container> &&
+                      has::end<Container> &&
+                      (! is::view<Container>) &&
+                      (! std::is_reference_v<Container>) &&
+                      (! std::is_trivially_copyable_v<ElementType>))
```

Comment:
worth to combine those concepts, no? :)

Reply:
I combined it in a protected bool variable template now